packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/inspections/impl/JavaDriverFieldCheckLinterInspectionTest.kt

```diff
@@ -56,7 +56,12 @@ public class Repository {
         psiFile: PsiFile,
         fixture: CodeInsightTestFixture,
     ) {
-        fixture.enableInspections(JavaDriverFieldCheckInspectionBridge::class.java)
+        fixture.file.virtualFile.putUserData(
+            MongoDbVirtualFileDataSourceProvider.Keys.attachedDialect,
+            JavaDriverDialect,
+        )
```

Comment:
Can we not use Fixture.specifyDialect here?

Reply:
Done! Also refactored to use the fixture.setupConnection method, that does also all the handling of adding the data source to the editor.